cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/lib/table/leveldb/LevelDBTableService.java

```diff
@@ -61,10 +61,8 @@
 
   private final ConcurrentMap<String, DB> tables = Maps.newConcurrentMap();
 
-  private static final LevelDBTableService SINGLETON = new LevelDBTableService();
-
   public static LevelDBTableService getInstance() {
-    return SINGLETON;
+    return new LevelDBTableService();
```

Comment:
It should not affect the non-preview scenario because when we start the StandaloneMain, we create new injector and we want the singleton instance of the `LevelDBTableService` from that injector.